tt_eager/tt_lib/csrc/operations/primary/module.hpp

```diff
@@ -742,6 +742,7 @@ void py_module(py::module& m_primary) {
         py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
         py::arg("output_dtype").noconvert() = std::nullopt,
         py::arg("in_place") = false,
+        py::arg("opt_output_tensor").noconvert() = std::nullopt,
```

Comment:
@eyonland does this arg need to explicitly be named `opt_*`?
Can it just be `output_tensor`?
None of our other optional args are named with `opt_*`.
pytoch api the optional output is also just `out`.

Reply:
Please make it output_tensor.

Reply:
I have changed it to output_tensor